assets/images/Leanne Gallagher _ Portfolio.html

```diff
@@ -0,0 +1,198 @@
+    <!-- Link to the CSS stylesheet that contains the aesthetics of the website -->
+    
+    <title>Leanne Gallagher | Portfolio</title>
+    <!-- icon for my webpage bar -->
+    
+    <!-- This is the link to the font I chose on googlefonts - 'Poppins' -->
+    
+    
+    
+  <link rel="stylesheet" href="./Leanne Gallagher _ Portfolio_files/style.css"><link rel="icon" href="http://127.0.0.1:5500/assets/images/icon.png" type="image/x-icon"><link rel="preconnect" href="https://fonts.googleapis.com/"><link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin=""><link href="./Leanne Gallagher _ Portfolio_files/css2" rel="stylesheet"></head>
+
+  <body>
+    <header class="header">
+      <img src="./Leanne Gallagher _ Portfolio_files/profile-img.png">
+      <h1>Leanne Gallagher</h1>
+      <!-- The navigation bar containing internal links to each important section of the webpage -->
+      <nav class="nav-container">
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#about">About Me</a>
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#portfolio">Portfolio</a>
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#contact">Contact</a>
+      </nav>
+    </header>
+
+    <main>
+      <!-- First main section of webpage with a short description about myself -->
+      <section id="about" class="about-me">
+        <h1>About Me</h1>
+        <p>
+          Hi, I'm Leanne. I'm 29 years old and I'm from Coventry. I am just
+          beginning my journey in coding and I am looking forward to where this
+          journey may take me. I am hardworking, inquisitive and passionate to
+          learn all I can.
+        </p>
+      </section>
+
+      <!-- Second section containing the portfolio and heading-->
+      <section id="portfolio" class="portfolio">
+        <h1>Portfolio</h1>
+
+        <!-- This is the section for the first project in the portfolio that has to span across the webpage -->
+        <section class="main-project-card">
+          <img src="./Leanne Gallagher _ Portfolio_files/main-project-screenshot.png">
+          <!-- This contains the information related to the first project including a short description and external link -->
+          <div class="main-project-card-info">
+            <h3>Semantic HTML Refactor</h3>
+            <p>
+              The objective of this project was to fix broken code, make the
+              webpage more accessible and refactor HTML semantics.
+            </p>
+            <a href="https://lenny-g.github.io/semantic-html-refactor/">view</a>
+          </div>
+        </section>
+        <!-- This is a section contained in the Portfolio section that holds the 6 other project cards and their placeholders to be edited at a later date -->
+        <section class="other-projects">
+          <div class="project-card">
+            <img src="./Leanne Gallagher _ Portfolio_files/250x250">
```

Comment:
As mentioned on the feedback, these placeholders are fine but they don't look the nicest. Consider replacing these with some stock images of code or something.